build/dsc.rake

```diff
@@ -75,9 +82,10 @@ eod
       FileUtils.rm_rf(Dir["#{dsc_resources_path_tmp}/**/.{gitattributes,gitignore,gitmodules}"])
       FileUtils.rm_rf(Dir["#{dsc_resources_path_tmp}/**/*.{pptx,docx,sln,cmd,xml,pssproj,pfx,html,txt,xlsm,csv,png,git,yml,md}"])
 
-      puts "Copying vendored resources from #{dsc_resources_path_tmp}/xDscResources to #{vendor_dsc_resources_path}"
-      FileUtils.cp_r "#{dsc_resources_path_tmp}/xDscResources/.", vendor_dsc_resources_path, :remove_destination => true
-      FileUtils.cp_r "#{dsc_resources_path_tmp}/xDscResources/.", dsc_resources_path
+      puts "Copying vendored resources from #{dsc_resources_path_tmp} to #{vendor_dsc_resources_path}"
+      FileUtils.cp_r "#{dsc_resources_path_tmp}/.", vendor_dsc_resources_path, :remove_destination => true
+      FileUtils.cp_r "#{dsc_resources_path_tmp}/.", dsc_resources_path
+      FileUtils.rm_rf "#{vendor_dsc_resources_path}/PSDesiredStateConfiguration"
```

Comment:
Also ensuring that those are explicitly deleted in case someone attempts to vendor those in.